docs/current_docs/configuration/cache.mdx

```diff
@@ -60,9 +60,7 @@ The default cache policy attempts to keep the long-term cache storage under 75%
 of the total disk, while also ensuring that at least 20% of the disk remains
 free for other applications and tools.
 
-The cache policy can be manually configured by editing the `engine.toml` file in the engine
-(which uses [BuildKit's syntax config](https://github.com/moby/buildkit/blob/v0.17.0/docs/buildkitd.toml.md?plain=1#L66-L82)).
-See the [Custom Runner](./custom-runner.mdx) docs for more information.
+The cache policy can be manually configured using the [engine config](./engine.mdx).
```

Comment:
This page (introduced in #8695) is still useful, but the actual configuration information is now all in `engine.mdx`. I think it makes sense to move this page somewhere else, but not entirely sure where, any suggestions @vikram-dagger?

Reply:
Hm, I'm not 100% sure about this.

I think having a separate caching page where we can talk about "how caching works" is important. Long term, I know we want to talk more about it (and it's probably going to get more complex), and we could even add information on how to optimize your pipelines for getting better cache results, with examples in different languages.

The `dagger query` examples to inspect cache and prune it are really useful, but I don't think they're related to configuration, and I wouldn't expect to find them under there.

WDYT about merging into https://docs.dagger.io/features/caching? Although, that section doesn't really have any code examples, so not sure about that. But it does feel like there's a lot of duplication between those.

Reply:
I would not put it in the features page, mainly because these seem like "operational" commands and the features pages are more "big picture"/"concepts" (side note: I'm not very confident that "caching best practices" in the features caching page is correct either, at some point this will move out).

WDYT of adding an entry to the FAQ page with these commands? e.g. `How do I inspect the Dagger cache?`

Reply:
JFYI I also thought of including these commands in https://docs.dagger.io/api/cache-volumes, but I think this is probably wrong as that page is specific to cache volumes only, not the layer cache. However let me know if you have a different opinion @jedevc

Reply:
> WDYT of adding an entry to the FAQ page with these commands? e.g. `How do I inspect the Dagger cache?` as a stopgap until we have a "proper" caching page?

I'm happy with that - we can add a redirect to there?

> JFYI I also thought of including these commands in [docs.dagger.io/api/cache-volumes](https://docs.dagger.io/api/cache-volumes), but I think this is probably wrong

Yeah, agreed - it's also that that page is relevant to the *usage* of cache volumes, while this is more about "operationally" using cache.

One *other* option would be to create a new API page on the core `engine` function, and the `Engine` type. Then, we could document the cache info there, as well as any other functionality we put under there. It's more specialized, and more geared towards operating a dagger engine.

Reply:
> One other option would be to create a new API page on the core engine function, and the Engine type. Then, we could document the cache info there, as well as any other functionality we put under there. It's more specialized, and more geared towards operating a dagger engine

I like this idea more than the FAQ, because this is more scalable to future changes. I'll add an atomic commit with a skeleton of this page so we can iterate further (assuming that's OK, lmk if not).

Reply:
Yup, that sounds perfect, thanks!

Reply:
Added page